### PR TITLE
dev_env -> example_env - mv -> cp

### PR DESCRIPTION
**Pull request by mmattbtw**
instead of advising to remove a file, creating more git changes, should probably just copy the example and rename it to .env

also from dev_env to example_env because it makes more sense imo :^)

### CODE REVIEW

**Comment by espeon** [comment]
id just move it to root directory and name it .env.example

File: apps/aqua/.example_env
Comment (espeon) [inline apps/aqua/.example_env:1]: move to /.env.example to better fit with existing best practices

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 To get started with this template, simply paste this command into your terminal:
 
 ```bash
-bun install && bun install -g turbo && mv apps/aqua/.dev_env apps/aqua/.env &&
+bun install && bun install -g turbo && cp apps/aqua/.example_env apps/aqua/.env &&
```
Comment (espeon) [inline README.md:6]: change location :)

Comment (mmattbtw) [reply README.md:6]: location of what

File: apps/aqua/.example_env
Comment (mmattbtw) [reply apps/aqua/.example_env:1]: e23efb6357c8d1adb72e3d669de350fa2c9dad54

**Review by kjloveless — COMMENTED** [review]
🐳🐳🐳

**Pull request merged**